src/backend/parser/cypher_expr.c

```diff
@@ -153,15 +153,15 @@ static Node *transform_cypher_expr_recurse(cypher_parsestate *cpstate,
     case T_NullTest:
     {
         NullTest *n = (NullTest *)expr;
-        NullTest *tranformed_expr = makeNode(NullTest);
+        NullTest *transformed_expr = makeNode(NullTest);
 
-        tranformed_expr->arg = (Expr *)transform_cypher_expr_recurse(cpstate,
+        transformed_expr->arg = (Expr *)transform_cypher_expr_recurse(cpstate,
                                                          (Node *)n->arg);
```

Comment:
I'n not sure about the whitespace here, but it doesn't seem to be particularly aligned, so I'm not touching it.